src/components/Header.js

```diff
@@ -2,13 +2,17 @@ import { Link } from 'react-router-dom';
 
 const Header = () => (
   <nav className="navbar">
-    <h1>Bookstore CMS</h1>
+    <h1 className="nav-header">Bookstore CMS</h1>
     <ul>
       <li>
-        <Link to="/">Books</Link>
+        <Link className="nav-link" to="/">
+          BOOKS
+        </Link>
       </li>
       <li>
-        <Link to="/categories">Categories</Link>
+        <Link className="nav-link" to="/categories">
+          CATEGORIES
+        </Link>
```

Comment:
- [ ] Kindly 😃 try to add the icon of the user at the right side of the header to make it nice like the zeplin design as you see you forgot to add the icon of user , so I suggest adding it to be nice like the original design 💯👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|   ![image](https://github.com/Jeddaa/bookstore/assets/83360636/c2bfb5d6-a4a7-403b-bc5f-05faafdb4360)   |    ![image](https://github.com/Jeddaa/bookstore/assets/83360636/2d97fd7f-5d83-41b0-a7f4-23bdfe544d7c) |